__main__.py

```diff
@@ -11,27 +11,45 @@
 from email.utils import make_msgid
 import optparse
 
-client_id_path = "./atefics/client.json"
+client_id_path = "/reg/g/pcds/epics-dev/nrw/ATEF/atefics/client.json"
```

Comment:
This is the json with the refresh token correct?  Even better might be allowing the user to add a path as a cli argument.  But doing that means you also probably want to parametrize the sender / receiver email addresses.  For now this is probably fine but food for thought.

Reply:
You might also put the sender/receiver email in the client json itself and just make that one big bundle of information.  Just thinking about how we might generalize this for other people to use.

Reply:
Maybe the more permanent recipients get added to the client.json and the we add an option to pass a list of addition recipients, and maybe an option to bypass the recipients in the client.json? I am not sure how many more features we should add do this until its no longer just a prototype.

Reply:
Yeah my suggestions are take it or leave it.  This one in particular is more of a future enhancement idea, but one that we'd need to make this more widely usable.